supplier_api_tester/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.0.6
+
+- bugfix: fix function `supplier_api_tester.v2.utils.adapters.get_api_error()` to handle `None` response.
```

Comment:
**question:** Based on the MR maybe a better description would be `to handle non-Json responses`?